web/frontend/src/components/slam-2d-render.vue

```diff
@@ -8,6 +8,42 @@ import { MapControls } from 'three/examples/jsm/controls/OrbitControls';
 import { PCDLoader } from 'three/examples/jsm/loaders/PCDLoader';
 import type { commonApi } from '@viamrobotics/sdk';
 
+/*
+ * // Leaving additional color map commented for if we want to change to a different scheme.
+ * generated with: https://waldyrious.net/viridis-palette-generator/
+ * more info: https://cran.r-project.org/web/packages/viridis/vignettes/intro-to-viridis.html
+ * // const colorMapViridis = [
```

Comment:
[q] Why are we storing the `colorMapViridis` here if we are using the grayscale one?

Reply:
I left the viridis colormap commented out so we could easily swap to it if after testing we found the greyscale color map did not perform the way we wanted

Reply:
the viridis color map is a separate mapping from the greyscale color map

Reply:
@EshaMaharishi just confirming we are alright having this alternative color schema as a comment in this file

Reply:
Generally I think it's better to remove unused code than leave it commented out, but I don't feel very strongly in this case if we think it's useful to have it available

Reply:
On second thought, I think we should remove it. It's not great to accumulate commented out code in a codebase. It'll always be in the git history if we need it.